packages/core/src/messenger.ts

```diff
@@ -60,6 +60,15 @@ export class JoinOrgInviteCompletedMessage extends Message<{ orgName: string; op
     }
 }
 
+export class FailedLoginAttemptMessage extends Message<{ srpId: string }> {
+    template = "failed-login-attempt";
+
+    get title() {
+        const appName = process.env.PL_APP_NAME;
+        return `${appName ? appName + " " : ""}Failed Login Attempt (SRP ID: ${this.data.srpId})`;
```

Comment:
Do you think this is helpful at all, or should we just not send any info?

Reply:
I'd leave out the SRP id since it'll only confuse users.

Reply:
Maybe add the location to the title. E.g.: `Failed Login Attempt from Munich, Germany`.